utils/cache.js

```diff
@@ -0,0 +1,113 @@
+const CACHEEXPIRYTIME = 2; // Default cache expriy time in Minutes
+const CACHESIZE = 10; // Default maximum size for cache pool (in Megabytes)
```

Comment:
1. Use underscore to separate words
2. Should we specify units in name? Reason: We are not using the defaults like bytes for size and milliseconds for time
```suggestion
const CACHE_EXPIRY_TIME_MIN = 2; // Default cache expriy time in Minutes
const CACHE_SIZE_MB = 10; // Default maximum size for cache pool (in Megabytes)
```